packages/astro/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@spotlightjs/astro",
+  "description": "Add spotlight to your Astro toolbar, for better debugging.",
+  "version": "0.0.0",
+  "type": "module",
+  "keywors": [
+    "astro",
+    "withastro",
+    "astro-component",
+    "analytics"
+  ],
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "main": "./src/index.tx",
+  "exports": {
+    ".": "./src/index.ts"
+  },
+  "dependencies": {
+    "@spotlightjs/core": "0.0.2",
+    "@spotlightjs/sidecar": "0.0.2",
```

shouldn't these two be set to
```suggestion
    "@spotlightjs/core": "workspace:*",
    "@spotlightjs/sidecar": "workspace:*",
```
?